scripts/Projector/flex_schedule_on.py

```diff
@@ -5,6 +5,6 @@
 
 def flex_schedule_on():
     ssh_connection = ssh.SSH(hostname, pi.username, pi.password)
-    ssh_connection.send_cmd("touch /tmp/flex")
+    ssh_connection.send_cmd("touch /home/pi/clock/public/flex-toggle")
     print("Done.")
```

Comment:
Ditto above

Reply:
resolved in [42bd97a](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/50/commits/42bd97abdfbdc7122db7036a34528b17eded5852)